Fix build failure:
xarfile.pyx:509:34: Empty declarator
https://stackoverflow.com/questions/8089412/empty-declarator-error-when-cythoning-a-class
I guess "type" is somehow a reserved word in Cython now.
--- xarfile.pyx.orig	2007-01-21 12:50:10.000000000 -0600
+++ xarfile.pyx	2018-11-04 06:20:24.000000000 -0600
@@ -506,10 +506,10 @@
 
 class XarInfo(object):
 
-    def __init__(self, name, type = '', mode = '', uid = '', user = '',
+    def __init__(self, name, _type = '', mode = '', uid = '', user = '',
             gid = '', group = '', atime = '', mtime = '', ctime = ''): 
         self.name = name
-        self.type = type
+        self.type = _type
         self.mode = mode
         self.uid = uid
         self.user = user
